iFixit/checkout-ui-extensions
# Add Analytics Code

## Overview

We want to add analytics tracking to our stores, specifically to handle analytics events that can no longer be thrown by our shopify stores once we move over to this extension. This code will run in shopify store's web pixels, as custom pixels, but will live in our codebase just so we can review changes/have a source of truth.

## QA

You should go through a checkout process and make sure all the events are firing on ga4 and piwik as expected. As of right now, do not expect any extra dimensional data, just the event data. Dimensions will be added later.

Connects: https://github.com/iFixit/checkout-ui-extensions/issues/34
